### ISSUE TITLE
Consistent CTD on turnover

### ISSUE THREAD

**Issue by skodkim** [LoneGazebo/Community-Patch-DLL#4044]
_1. Mod version (i.e Date - 4/23):_
2/27

_2. Mod list (if using Vox Populi only, leave blank):_
(1) Community Patch
(2) Community Balance Overhaul
(3) CSD for CBP
(4) Civ IV Diplomatic Features
(5) More Luxuries
(6a) Community Balance Overhaul - Compatibility Files (EUI)
(7) Community Events (v 4)
Calypso's Colored Religious Icons (Basic) (v 4)
CivWillard for BNW (v 2)
CPP CivPack (v 28)
CPP TweakPack (v 49)
CPP WondersPack (v 27)
Cultural Capitals (Brave New World Edition) (v 16_5)
FlagPromotions (v 7)
InfoAddict (v 22)
Ingame Editor (v 39)
JFD's Cultural Diversity (Core) (v 13)
JFD's Rise to Power (v 34)
LiveTuner Panels (v 1)
Sukritacts Events and Decisions (v 3)
TerrainMod (v 7)
UI - Gold Alerts (v 10)
UI - Map Pins (v 17)
UI - Overlay Barbarians (v 8)
UI - Promotion Tree (v 15)
UI - Summary Specialists (v 11)
UI - Trade Opportunities (v 28)
UI - Unit List Enhancements (v 11)
Whowards PickNMix Mods (v1)
Wonder Race (v 7)

_3. Error description:_ 
I have a consistent CTD on turnover after turn 180 epic speed). Seems to be around the time where the Inca is doing their turn (Player before is Vikings and I think they finish ok).
[CvMiniDump_2.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/1799200/CvMiniDump_2.zip)

I use a lot of mods (but have done so for a while and have not had ctd experiences with these before) so it may not be Vox Populi retated. Log files seem to be ok and I can't open the dump file however so I have no clue as to what is going on.

_4. Steps to reproduce (optional):_

---------------------------
Supporting information:
Please note that you can attach .zip files by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

1. Log files (always attach your Logs folder, located at My Documents/My Games/Sid Meier's Civilization 5. Make sure you have enabled logging before experiencing an error! Go here to find out how: http://forums.civfanatics.com/showthread.php?t=487482):

[Logs.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/1799195/Logs.zip)

2. Save game (always attach a save that was made a turn before the error; located at My Documents/My Games/Sid Meier's Civilization 5/ModdedSaves):

3. CvMiniDump.dmp file (attach if experiencing a game crash. Located at Program Files/Steam/steamapps/common/Sid Meier's Civilization V):

[CvMiniDump.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/1799198/CvMiniDump.zip)

4. Screenshots (optional):

**Comment by Vyyt02** [comment]
Did you install the JFD's Rise to Power via the JFDLC? Sorry, I have not had a chance to try these by myself, so maybe I don't know what I am talking about, but the install process changed recently. I updated the compatibility list on Civfanatics based on instructions from JFD - maybe worth checking them.

**Comment by skodkim** [comment]
No, I'm running the last version that came out before before it became a party of JFDLC. JFD pushed it in the RtP thread a while ago.

\Skodkim

⁣Sendt af BlueMail ​

Den 17. mar. 2018 20.27, fra 20.27, Vyyt <notifications@github.com> skrev:

**Comment by LoneGazebo** [comment]
So many other mods....there's not much I can do with this,  the CTD doesn't crash to the DLL so it's out of my hands...
